akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ClusterSharding.scala

```diff
@@ -749,15 +748,16 @@ private[akka] class ClusterShardingGuardian extends Actor {
                 ShardCoordinator.props(typeName, settings, allocationStrategy)
               else
                 ShardCoordinator.props(typeName, settings, allocationStrategy, rep, majorityMinCap)
-            val singletonProps = BackoffSupervisor
-              .props(
-                childProps = coordinatorProps,
-                childName = "coordinator",
-                minBackoff = coordinatorFailureBackoff,
-                maxBackoff = coordinatorFailureBackoff * 5,
-                randomFactor = 0.2,
-                maxNrOfRetries = -1)
-              .withDeploy(Deploy.local)
+            val singletonProps =
+              BackoffOpts
+                .onFailure(
```

Comment:
This was a change in behavior, which caused https://github.com/akka/akka/issues/26724 to fail. I think the old behavior corresponds to `onStop` (which may happen for persist failures). I'm fixing.

Reply:
dang, sorry!